src/IO/AssimpLoader/AssimpFileLoader.cpp

```diff
@@ -70,14 +70,14 @@ FileData* AssimpFileLoader::loadFile( const std::string& filename ) {
     // failes with Light only. If one need to load file with only skeleton or animation, this must
     // be changed. Note that loading only skeletons or animations from a file is not allowed in
     // Radium For now, Skeleton must be loaded from a file with meshes
-    if ( scene->mFlags & AI_SCENE_FLAGS_INCOMPLETE )
-    {
-        LOG( logWARNING ) << " ai scene is incomplete, just try to load lights ..";
-        AssimpLightDataLoader lightLoader( Core::Utils::getDirName( filename ),
-                                           fileData->isVerbose() );
-        lightLoader.loadData( scene, fileData->m_lightData );
-    }
-    else
+//    if ( scene->mFlags & AI_SCENE_FLAGS_INCOMPLETE )
+//    {
+//        LOG( logWARNING ) << " ai scene is incomplete, just try to load lights ..";
+//        AssimpLightDataLoader lightLoader( Core::Utils::getDirName( filename ),
+//                                           fileData->isVerbose() );
+//        lightLoader.loadData( scene, fileData->m_lightData );
+//    }
+//    else
```

Comment:
Unexpected deadcode

Reply:
I intended to keep this code since it is related to an issue on assimp (see comment above) which has been closed with no useful response.
Want me to remove it?